packages/react-notion-custom/src/lib/index.css

```diff
@@ -564,6 +564,31 @@
   padding: 3px 2px;
 }
 
+.notion-list-bulleted {
+  list-style-type: none;
+  padding-left: 0;
+}
+
```

Comment:
p2: Please place the CSS at the very bottom to minimize merge conflicts.

Reply:
> p2: Please place the CSS at the very bottom to minimize merge conflicts.

I've moved the CSS to the very bottom of the file as requested.